daprdocs/content/en/developing-applications/integrations/open-service-mesh.md

```diff
@@ -0,0 +1,27 @@
+---
+type: docs
+title: "Running Dapr and Open Service Mesh together"
+linkTitle: "Open Service Mesh"
+weight: 4000
+description: "Learn how to run both Open Service Mesh and Dapr on the same Kubernetes cluster"
+---
+
+## Overview
+
+[Open Service Mesh (OSM)](https://openservicemesh.io/) is a lightweight, extensible, Cloud Native service mesh that allows users to uniformly manage, secure, and get out-of-the-box observability features for highly dynamic microservice environments.
+
+{{< button text="Learn more" link="https://openservicemesh.io/" >}}
+
+## Dapr integration
+
+Users are able to leverage both OSM SMI traffic policies and Dapr capabilities on the same Kubernetes cluster. Visit [this guide](https://docs.openservicemesh.io/docs/integrations/demo_dapr/) to get started.
+
+{{< button text="Deploy OSM and Dapr" link="https://docs.openservicemesh.io/docs/integrations/demo_dapr/" >}}
+
+## Example
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/LSYyTL0nS8Y?start=1916" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
When you say example here, I do not see a description of the example there,, other than "YouTube video player". Is this supposed to have a description?

Reply:
Added a description